jsk_pcl_ros/test/test_octomap_contact.test

```diff
@@ -0,0 +1,35 @@
+<launch>
+
+  <include file="$(find jsk_pcl_ros)/sample/sample_octomap_contact.launch">
+    <arg name="rviz" value="false" />
+  </include>
+
+  <test test-name="test_octomap_contact"
+        name="test_octomap_contact"
+        pkg="jsk_tools" type="test_topic_published.py"
+        time-limit="30" retry="3">
+    <rosparam>
+      topic_0: /occupied_cells_vis_array
+      timeout_0: 30
+      topic_1: /free_cells_vis_array
+      timeout_1: 30
+      topic_2: /unknown_cells_vis_array
+      timeout_2: 30
+      topic_3: /frontier_cells_vis_array
```

Comment:
`/frontier_cells_vis_array` is advertised but somehow not published yet.

Reply:
I am investigating it.